4_arm.ipynb

```diff
@@ -194,13 +348,37 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "In addition, in \"Motion generation using forward kinematics\", only the HSR arm was used. However,  we cannot grab distant objects by moving the arm alone.\n",
+    "\n",
+    "HSR is a \"mobile manipulator\" that has the arm mounted on the move base. We will show the combined control using the move base and the arm from now on."
```

Comment:
"The HSR is a \"mobile manipulator\" that has the arm mounted on the moving base. We will show the combined control using the moving base and the arm from now on."